code/datums/helper_datums/teleport.dm

```diff
@@ -141,6 +142,12 @@
 		destturf = safepick(posturfs)
 	else
 		destturf = get_turf(destination)
+	
+	// Make sure the target tile does not contain a teleporter on it
+	for(var/teleporter_type in blacklisted)
+		var/teleporters = destturf.search_contents_for(teleporter_type)
+		if(length(teleporters))
+			return 0
```

Comment:
```suggestion
			return FALSE
```

Reply:
I figured it was probably fine since the rest of the file does this.

Reply:
Probably means all the code could get a little update, but it's fine either way tbh

Reply:
Adressed in the entire file.